CHANGELOG.md

```diff
@@ -116,6 +116,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated the access to the attribute `IndexBatchSamplerWrapper.batch_indices` in favor of `IndexBatchSamplerWrapper.seen_batch_indices` ([#10870](https://github.com/PyTorchLightning/pytorch-lightning/pull/10870))
 
 
+- Deprecated `Trainer.call_hook` ([#10979](https://github.com/PyTorchLightning/pytorch-lightning/pull/10979))
```

Comment:
can you make it more verbose?
```
Deprecated `foo` in favour of `bar`
```

Reply:
the thing is we are not really offering the user any alternative functionality

Reply:
yep, I know but just for our own reference or some new contributor.. in case someone goes back and looks at that, it got split into which 4 different methods.

Reply:
ok sure